src/main.rs

```diff
@@ -153,7 +162,24 @@ fn create_dirs(cx: &Context) -> Result<()> {
     Ok(())
 }
 
-fn set_env(cx: &Context, cmd: &mut ProcessBuilder) {
+trait EnvTarget {
+    fn set(&mut self, key: &str, value: &str);
+}
+
+impl EnvTarget for ProcessBuilder {
+    fn set(&mut self, key: &str, value: &str) {
+        self.env(key, value);
+    }
+}
+
+impl EnvTarget for std::io::Stdout {
+    fn set(&mut self, key: &str, value: &str) {
+        use std::io::Write;
+        writeln!(self, r#"{}="{}""#, key, value).expect("failed to write to stdout");
```

Comment:
It may be useful to have an option to output environment variables with the `export ` prefix.

It allows us to save the output to a file and then `source` it as-is. like the following:
```sh
cargo llvm-cov show-env --export-prefix > ./env 
source ./env
```